app/routes.js

```diff
@@ -196,6 +196,7 @@ export default function createRoutes(store) {
     }, {
       path: profilePage(),
       name: 'Profile',
+      onEnter: requireAuth,
```

Comment:
This looks redundant with for me with:
https://github.com/zmora-agh/zmora-ui/pull/233/files#diff-71bee332c774afdddad7edcfe3762201R21